arch/x86/include/asm/kvm_pkvm.h

```diff
@@ -56,6 +56,20 @@ enum pkvm_fn {
 	__pkvm__enable_virtualization_cpu,
 	__pkvm__disable_virtualization_cpu,
 	__pkvm__vm_init,
+
+	/* Below PV interfaces should use kvm_call_pkvm_inout */
+	PKVM_FIRST_INOUT_PV_INTERFACE,
+};
+
+#define PKVM_FN_DATA_LEN		4
```

Comment:
nit: maybe PKVM_FN_ARG_NUM or something like that?

even nitter: reduce distance (spaces) between `PKVM_FN_DATA_LEN` and `4`? :)